R/createReport.R

```diff
@@ -79,6 +79,7 @@ createReport = function(txt_folder = NULL, mztab_file = NULL, yaml_obj = list(),
     txt_files$msms = "msms.txt"
     txt_files$msmsScan = "msmsScans.txt"
     txt_files$mqpar = "mqpar.xml"
+    txt_files$cv = "qc_cv.obo"
```

Comment:
the `obo` is not a MaxQuant file and should be defined outside of `txt_files`.
e.g. `cv_obo_file = "qc_cv.obo"`